### PR TITLE
[IAC-1735] - update spec tests

### PR DESCRIPTION
**Pull request by adrianiurca**
The tests related to classes or defines which are using `ensure_packages` function from `stdlib` needs to be updated accordingly to this PR https://github.com/puppetlabs/puppetlabs-stdlib/pull/1196

### CODE REVIEW

File: spec/spec_helper_local.rb [hunk 1]
```diff
@@ -44,6 +44,10 @@ def param(type, title, param)
   param_value(catalogue, type, title, param)
 end
 
+def postgresql_version
+  '9.6'
+end
+
```
Comment (smortex) [inline spec/spec_helper_local.rb:50]: This function does not seem to be used in the rest of the changes of this PR.  Was it added by mistake?

**Review by smortex — COMMENTED** [review]
LGTM, but I am confused about this new function :thinking:

**Review by ekohl — APPROVED** [review]
I removed the additional method. Will squash on merge.

**Comment by ekohl** [comment]
Ah, I now see this was added because it's actually used in `spec/unit/defines/server/default_privileges_spec.rb`. https://github.com/puppetlabs/puppetlabs-postgresql/pull/1267/files#r689125725 was wrong to begin with.

**Comment by ekohl** [comment]
I opened https://github.com/puppetlabs/puppetlabs-postgresql/pull/1291 which takes most of this commit, but fixes the broken unit test separately with let blocks.

**Pull request closed without merging**